File: DesignPatternsExercises/java/ChainOfResponsibilityDesignPattern/Solution/ConfigurationSupport.java
// public class ConfigurationSupport extends __________(6)__________ {
public class ConfigurationSupport extends SupportChain {
	@Override
	public void handleIssue(IssueType issueType) {
		if(issueType == IssueType.Configuration) {
			System.out.println("Configuration support team will contact you shortly");
		} else {
			// __________(7)__________(issueType);
			nextSupportLink.handleIssue(issueType);
		}
	}
}

File: DesignPatternsExercises/ruby/chain_of_responsibility_design_pattern/solution/configuration_support.rb
require_relative 'support_chain'

class ConfigurationSupport < SupportChain
  def handle_issue(issue_type)
    if issue_type == :configuration
      puts 'Configuration support team will contact you shortly'
    else
      # @next_support_link.__________(1)__________(issue_type)
      @next_support_link.handle_issue(issue_type)
    end
  end
end
